ale_linters/ruby/brakeman.vim

```diff
@@ -0,0 +1,83 @@
+" Author: Eddie Lebow https://github.com/elebow
+" Description: Brakeman, a static analyzer for Rails security
+
+function! ale_linters#ruby#brakeman#Handle(buffer, lines) abort
+    let l:result = json_decode(join(a:lines, ''))
+
+    let l:output = []
+
+    for l:warning in l:result.warnings
+        if !s:MatchFileWithoutRailsRoot(l:warning.file, a:buffer)
```

Comment:
I need to write that function for matching filenames against a buffer. This is a common pattern.